papers/whitepaper/currency.tex

```diff
@@ -1,44 +1,41 @@
- approach..." paper that is a fee structure, not usefulness of a token.}
-
- In this section, we provide some reflections on the nature of the native Ergo token. For starters, we note that
- any currency has three main functions: a medium of exchange, a unit of account, and a store of value.
-
- Bitcoin, being historically the first digitally scarce asset, is perfect as store-of-value. It is even better than gold under certain assumptions~(such as the SHA-256 hash function not being broken, and a majority of miners not willing to destroy Bitcoin), as emission is limited and known in advance. However, being the perfect store-of-value also means that it is not so good as medium-of-exchange. In particular, if one knows that Bitcoin is indeed the best tool to store value in the long term, he would use fiat whenever possible to collect more bitcoins.
-
- On the other hand, Ethereum is not just a currency, but a utility token used to pay for computations over the
-``decentralized world computer'' (or ``fully replicated programmable calculator''). However, Ethereum is not good as
- store-of-value, as emission is endless, and, historically, can be changed easily by the community along with critical system parameters.
-
- Ergo combines best from these two top blockchains. Emission is predefined and limited. Furthermore, it will be finished within
- just ten years. The system assumptions are set in stone with a precisely defined {\em social contract}~\ref{sec:social}. Also,
- Ergo is a utility token used to pay storage rent. This storage rent makes the system more stable. Lastly, Ergo is suitable for building monetary systems on top of it with properties different from the Ergo native token itself. However, participating in such systems would require using the Ergo native token as well in order to pay storage rent.
-
-
-\subsection{Currency And Emission}
 \label{sec:currency}
 
-The native currency of \Ergo{} platform is the \Erg{} token, whose unique property is
-that it is the only currency for paying storage rent in \Ergo{}~(see Section~\ref{sec:survivability} for more details).
-One \Erg{} is divisible to up to $10^9$ smaller units, called \nanoErg{}s.
-
-All \Erg{} tokens that will ever circulate in the system are present in
-the initial state and are divided into 3 parts~(boxes):
-
-\knote{no any notion of a box was introduced before, the introduction is sudden}
-\dnote{I propose to add section about Ergo state before this one and introduce box notion in it}
+As most of the blockchain systems, Ergo platform have its native token which is
+called \Erg{} and is divisible to up to $10^9$ smallest units, \nanoErg{}s.
+\Erg{} provides stability to Ergo platform by several meaningful applications.
+
+During the initial phase of Ergo live, miners will receive the reward in \Erg{}s created
+out of thin air according to a predefined and hard-coded token emission schedule
```

Comment:
Not out of thin air, but out of a miners rewards box protected by an emission script.